src/components/slider/Slider.js

```diff
@@ -0,0 +1,250 @@
+import React, { Component } from "react";
+import Slider from "react-slick";
+import Icon from "../Icons";
+import IMG1 from "../../assets/img/Tian.jpg";
+import IMG2 from "../../assets/img/katro.jpg";
+import IMG3 from "../../assets/img/Melodist.png";
+import IMG4 from "../../assets/img/cutting.jpg";
+import './_slider.scss';
+import "slick-carousel/slick/slick.css";
+import "slick-carousel/slick/slick-theme.css";
+
+function SampleNextArrow(props) {
+    const { onClick } = props;
+    return (
+        <div className="slider__buttons">
+            <button className="btn btn__arrow btn_next js-btn_next" onClick={onClick}> 
+                <Icon className="btn__icon" name="arrow" />
+            </button>
+        </div>
+        
+    );
+}
+
+function SamplePrevArrow(props) {
+    const { onClick } = props;
+    return (
+        <div className="slider__buttons">
+            <button className="btn btn__arrow btn__prev js-btn_prev" onClick={onClick}>
+                <Icon className="btn__icon" name="arrow" />
+            </button>
+        </div>
+        
+    );
+}
+
+export default class AsNavFor extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            nav1: null,
+            nav2: null
+        };
+    }
+
+    componentDidMount() {
+        this.setState({
+            nav1: this.slider1,
+            nav2: this.slider2
+        });
+    }
```

Comment:
you can try to rewrite into hooks. check `useEffect` doc. you can use it instead of `componentDidMount`.